doc/api/util.md

```diff
@@ -183,6 +183,10 @@ property take precedence over `--trace-deprecation` and
 <!-- YAML
 added: v0.5.3
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/17907
+    description: The `%o` specifiers `depth` option will now fall back to the
```

Comment:
Is this a typo (`specifier’s`)?

Reply:
I guess so. @vsemozhetbyt @Trott can you verify this? :-)

Reply:
I am not a native speaker, but it seems for me that `` `%o` specifier's `depth` option `` or `` `%o` specifier `depth` option `` would be OK?

Reply:
Yes, there should be an apostrophe: `specifier's`